.github/workflows/test.yml

```diff
@@ -31,7 +31,14 @@ jobs:
           bash tools/ci/build_tgui.sh
           bash tools/ci/check_grep.sh
           python3 tools/mapmerge2/dmi.py --test
-          ~/dreamchecker
+          ~/dreamchecker > ${GITHUB_WORKSPACE}/output-annotations.txt 2>&1
+
+      - name: Annotate Lints
+        uses: yogstation13/DreamAnnotate@v1
+        if: always()
```

Comment:
I know Yog has this, but I'm not really sure what the point of it is. The point is to continue if something else fails, I think.

@alexkar598 Any chance you can shed light on this?

Reply:
in my experience with github actions it likes to halt on fails yes, I would imagine this is just to ensure this step always runs even if the last step failed, because you want the annotations of the failure

Reply:
Makes sense to me :+1:

Reply:
In an ideal world, we could use conditionals to only run when the linter step failed but I didnt get it to work and I was already pretty done with the project so I hacked that in. It works but it's not ideal.

Reply:
In an ideal world, we could use conditionals to only run when the linter step failed but I didnt get it to work and I was already pretty done with the project so I hacked that in. It works but it's not ideal.